Resources/Textures/Clothing/Uniforms/Jumpsuit/overalls_cargo.rsi/meta.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at commit https://github.com/tgstation/tgstation/commit/ac792e3226d48e5b3aaccff165ce100f7636a040, monkey made by brainfood1183 (github) for ss14, rehued by lzk228 (github)",
```

Comment:
You appear to have copypasted this as Swept wasn't involved in these sprites at all. Also, this is a commit for the mailman jumpsuit.

Reply:
i copied that from botany overalls copyright and added rehued part, i suppose Swept was involved in botany overalls sprite